<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="43800" started-at="2017-05-22T15:12:49Z" finished-at="2017-05-22T15:13:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="43800" started-at="2017-05-22T15:12:49Z" finished-at="2017-05-22T15:13:33Z">
      <class name="msbuddy.msbuddytests">
        <test-method status="PASS" signature="testInstallAppLaunch()[pri:0, instance:msbuddy.msbuddytests@61832929]" name="testInstallAppLaunch" duration-ms="20201" started-at="2017-05-22T08:12:49Z" finished-at="2017-05-22T08:13:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInstallAppLaunch -->
        <test-method status="PASS" signature="testWelcomeScreens()[pri:0, instance:msbuddy.msbuddytests@61832929]" name="testWelcomeScreens" duration-ms="13684" started-at="2017-05-22T08:13:09Z" finished-at="2017-05-22T08:13:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWelcomeScreens -->
        <test-method status="FAIL" signature="testEmailLogin()[pri:0, instance:msbuddy.msbuddytests@61832929]" name="testEmailLogin" duration-ms="5016" started-at="2017-05-22T08:13:23Z" finished-at="2017-05-22T08:13:28Z">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at msbuddy.msbuddytests.testEmailLogin(msbuddytests.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmailLogin -->
        <test-method status="FAIL" signature="testTodaysMatch()[pri:0, instance:msbuddy.msbuddytests@61832929]" name="testTodaysMatch" duration-ms="4015" started-at="2017-05-22T08:13:28Z" finished-at="2017-05-22T08:13:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<May 22> but was:<>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<May 22> but was:<>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at msbuddy.msbuddytests.testTodaysMatch(msbuddytests.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTodaysMatch -->
        <test-method status="PASS" signature="quit()[pri:0, instance:msbuddy.msbuddytests@61832929]" name="quit" is-config="true" duration-ms="861" started-at="2017-05-22T08:13:32Z" finished-at="2017-05-22T08:13:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- msbuddy.msbuddytests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
